packages/cadl-ranch-coverage-sdk/src/client.ts

```diff
@@ -126,3 +160,14 @@ async function readJsonBlob<T>(blobClient: BlockBlobClient): Promise<T> {
   const content = await body?.text();
   return content ? JSON.parse(content) : undefined;
 }
+
+async function readJsonBlobFromStream<T>(blobClient: BlockBlobClient): Promise<T> {
```

Comment:
Why is this needed? Isn't the method above already doing that? If it is better why not changing it above

Reply:
```js
/** Contains response data for the download operation. */
export declare type BlobDownloadResponseModel = BlobDownloadHeaders & {
    /**
     * BROWSER ONLY
     *
     * The response body as a browser Blob.
     * Always `undefined` in node.js.
     */
    blobBody?: Promise<Blob>;
    /**
     * NODEJS ONLY
     *
     * The response body as a node.js Readable stream.
     * Always `undefined` in the browser.
     */
    readableStreamBody?: NodeJS.ReadableStream;
    /** The underlying HTTP response. */
    _response: coreHttp.HttpResponse & {
        /** The parsed HTTP response headers. */
        parsedHeaders: BlobDownloadHeaders;
    };
};```

Above method is calling blobBody, which is browser only, so it could be used when we download from website. This method is calling readableStreamBody, which is nodejs only, so it is used when we upload index from command.

Reply:
But in my latest code, I don't need this anymore.